extensions/amp-video/0.1/amp-video.js

```diff
@@ -575,6 +576,33 @@ export class AmpVideo extends AMP.BaseElement {
     return source;
   }
 
+  /**
+   * @private
+   * @return {boolean}
+   */
+  hasAnyBitrateSources_() {
+    return (
+      !!this.element.querySelector('source[data-bitrate]') ||
+      this.hasAnyCachedSources_()
+    );
```

Comment:
If it has cachedSources it will generate the bitrate sources in the buildCallback, so we can assume it has bitrate sources if it has cached sources